deploy/gcp/tidbclusters.tf

```diff
@@ -1,19 +1,14 @@
 data "local_file" "kubeconfig" {
   depends_on = [module.tidb-operator.cluster_id]
-  filename   = module.tidb-operator.kubeconfig_path
-}
-resource "local_file" "kubeconfig" {
-  depends_on = [module.tidb-operator.cluster_id]
-  content    = data.local_file.kubeconfig.content
-  filename   = module.tidb-operator.kubeconfig_path
+  filename   = local.kubeconfig
```

Comment:
resource `local_file.kubeconfig` is not needed because it simply creates the same file again with the contents in it